README.md

```diff
@@ -1,5 +1,7 @@
 # studio-frontend
 [![codecov](https://codecov.io/gh/edx/studio-frontend/branch/master/graph/badge.svg)](https://codecov.io/gh/edx/studio-frontend)
+[![Build Status](https://travis-ci.org/edx/studio-frontend.svg?branch=master)](https://travis-ci.org/edx/studio-frontend)
+[![npm package](https://badge.fury.io/js/%40edx%2Fstudio-frontend.svg)](https://badge.fury.io/js/%40edx%2Fstudio-frontend)
```

Comment:
Maybe throw in some `npm` downloads?

[![npm](https://img.shields.io/npm/dt/@edx/studio-frontend.svg)](https://www.npmjs.com/package/@edx/studio-frontend)